docs/standard/base-types/standard-numeric-format-strings.md

```diff
@@ -20,6 +20,9 @@ author: "rpetrusha"
 ms.author: "ronpet"
 ---
 # Standard Numeric Format Strings
+
+[!INCLUDE[interactive-note](~/includes/csharp-interactive-note.md)]
```

Comment:
adding this as the first thing in the topic might hurt SEO. Maybe move this to where the samples start?
Also, should the note discuss culture as well? Things like this could confuse people (where text, snippet and interactive don't match): 
![image](https://user-images.githubusercontent.com/12971179/41019836-3ff38318-6915-11e8-852e-51538851304d.png)